hrabsence/api/v3/Activity/Getabsences.php

```diff
@@ -127,6 +127,7 @@ function civicrm_api3_activity_getabsences($params) {
   $entity = _civicrm_api3_get_BAO(__FUNCTION__);
   $bao = CRM_Core_DAO::executeQuery($select->toSQL(), array(), TRUE, 'CRM_Activity_BAO_Activity');
   $activities = _civicrm_api3_dao_to_array($bao, $params, FALSE, $entity, FALSE);
-  $activities = _civicrm_api3_activity_get_formatResult($params, $activities);
+  $options = _civicrm_api3_get_options_from_params($params, FALSE, $entity, 'get');
```

Comment:
In this case won't `$entity` be the name of the BAO class? In other examples where this is called I see it using the short entity name, e.g.

```
_civicrm_api3_get_options_from_params($params, TRUE, 'contribution', 'get');
_civicrm_api3_get_options_from_params($params, TRUE, 'Membership', 'get');
_civicrm_api3_get_options_from_params($params, TRUE, 'ReportTemplate', 'get');
```